### PR TITLE
Add option to send multiple header ini (#954?)

### PR DESCRIPTION
**Pull request by rejoe2**
Offering the option to set multiple header init. 
forum#7454, post#6
Perhaps changing default also to three could be a good idea...

May also help wrt to this PR: 
https://github.com/mysensors/MySensors/issues/954

### CODE REVIEW

**Comment by mfalkvidd** [comment]
@rejoe2 would you mind doing the following, to make this feature easier for users to find and use?
- [ ] add to keywords.txt to get syntax highlightning in the Arduino IDE
- [ ] add to MyConfig.h with doxygen comments so this feature gets added to the [documentation](https://www.mysensors.org/apidocs-beta/group__RS485SettingGrpPub.html#gaa7614b7975eccb4f40036dfd007ad885). Use the [existing rs485 settings](https://github.com/mysensors/MySensors/blob/aabddc841cd526ffd716f4e467b4164b2e267a95/MyConfig.h#L203) as template.

The easiest way would probably be to move the _if !defined_ chunk to MyConfig.h and add doxygen comments.

**Comment by fallberg** [comment]
@rejoe2 as long as you keep adding commits and don't amend your failing commits, gitler will never approve this pr. Instructions for amending is mailed to you (or you can click the details link on the gitler toll gate status here)

**Comment by rejoe2** [comment]
Thanks for the feedback. As I'm just a hobbyist these methods and tools are new to me, I didn't realise my commits made online in the repo had directly gone to the main repo. I'll try to read and understand the documents and then try again.

**Comment by fallberg** [comment]
They go into your development branch on your fork of the main repo. And since you have filed a pull request from your development branch to the development branch of  the main repo, they show up here. Don't worry, the worst thing that can happen is that you break your forked repo, and if this is the only change, you can just remove the fork and create another one and apply the same change again.

File: keywords.txt [hunk 1]
```diff
@@ -177,6 +177,7 @@ MY_RS485_BAUD_RATE	LITERAL1
 MY_RS485_MAX_MESSAGE_LENGTH	LITERAL1
 MY_RS485_HWSERIAL	LITERAL1
 MY_RS485_DE_PIN	LITERAL1
+MY_RS485_SOH_COUNT LITERAL1
```
Comment (mfalkvidd) [inline keywords.txt]: The Arduino IDE is really stupid. It requires a **tab** character between "MY_RS485_SOH_COUNT" and "LITERAL1". At the moment you have a space character. Could you fix this?

**Comment by mfalkvidd** [comment]
@rejoe2 about the Gitler error: Gitler doesn't like human-readable URLs (at least not if they are long). You need to obfuscate the URL by using a URL shortening service like bit.ly or goo.gl. See https://github.com/mysensors/MySensors/commit/a23ab14494d28f90b019ea3c651dd61d50382e63 for how I did it when I stumbled into this problem.

**Comment by rejoe2** [comment]
Thanks for your hints. 
Now, I committed a second, cleaned-up version (#1014) as there seems not to exist the possibility to change remarks.

Thanks a lot also for supporting this cool project!

**Pull request closed without merging**